src/Stratis.Bitcoin.Features.PoA/PoAMiner.cs

```diff
@@ -192,7 +192,7 @@ private async Task CreateBlocksAsync()
                     // The purpose of bootstrap mode is to kickstart the network when the last mined block is very old, which would normally put the node in IBD and inhibit mining.
                     // There is therefore no point keeping this mode enabled once this node has mined successfully.
                     // Additionally, keeping it enabled may result in network splits if this node becomes disconnected from its peers for a prolonged period.
-                    if (this.settings.BootstrappingMode)
+                    if (this.settings.BootstrappingMode && !this.settings.DevMode)
```

Comment:
How do you bootstrap network when using dev mode?

Reply:
@StratisIain informed me that idea here is the node should be able to mine and validate smart contracts without it being connected to any other nodes.